### ISSUE TITLE
Implementing the Supercomputer module

### ISSUE THREAD

**Issue by robinlombaert** [IvS-KULeuven/ComboCode#8]
With the conversion of the old supercomputer in Leuven, VIC3, to the new one, called ThinKing, the module Vic.py has to be converted to be applicable to ThinKing. 

This is a good opportunity to expand the functionality of Vic.py so that it can be applied to multiple supercomputers (identified by an address, job queuing syntax, login username, etc). The idea is to create a module that provides all the tools that are needed regardless of which supercomputer is being accessed, such as tracking running jobs, checking if the code was ran successfully, adding results to databases, etc. Additional modules for each supercomputer then add the specific functionality required to access it and run jobs. 

In other words:
cc/managers/Supercomputer.py 
cc/managers/sc/ThinKing.py
cc/managers/sc/Slurm.py
etc.

An overview markdown file was created for the conversion from VIC3 to ThinKing specifically, located at:
<a href="https://github.com/IvS-KULeuven/ComboCode/blob/master/cc/managers/supercomputer_overview.md"> cc/managers/supercomputer_overview.md.

**Comment by robinlombaert** [comment]
Moved to the main ComboCode repo.
